integrationTests/singleShard/block/executingMiniblocksSc_test.go

```diff
@@ -0,0 +1,342 @@
+package block
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"io/ioutil"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/core/logger"
+	"github.com/ElrondNetwork/elrond-go/data/state"
+	"github.com/ElrondNetwork/elrond-go/data/transaction"
+	"github.com/ElrondNetwork/elrond-go/integrationTests"
+	"github.com/ElrondNetwork/elrond-go/sharding"
+	"github.com/stretchr/testify/assert"
+)
+
+var agarioFile = "agarioV2.hex"
+var stepDelay = time.Second
+
+func TestShouldProcessBlocksWithScTxsJoinAndRewardTheOwner(t *testing.T) {
```

Comment:
function too long

Reply:
reduced to 47 chars

Reply:
not the function name. but the number of the lines in the function